openedx_webhooks/lib/github/test/models/test_github_webhook_event.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import arrow
+import pytest
+
+from openedx_webhooks.lib.github.models import GithubWebHookEvent
+
+
+@pytest.fixture
+def issue_comment():
+    payload = {
+        'action': 'edited',
+        'issue': {
+            'html_url': 'https://example.com/issue/1',
+            'updated_at': '2016-10-24T18:53:10Z',
+        },
+        'sender': {
+            'login': 'issue-sender',
+        },
+    }
+    return GithubWebHookEvent('issue_comment', payload)
+
+
+@pytest.fixture
+def pull_request_review():
+    payload = {
+        'action': 'submitted',
+        'pull_request': {
+            'html_url': 'https://example.com/pull/1',
+            'updated_at': '2016-10-24T18:19:44Z',
+        },
+        'sender': {
+            'login': 'pr-sender',
+        },
+    }
+    return GithubWebHookEvent('pull_request_review', payload)
+
+
+class TestGithubWebHookEventResource:
+    def test_issue(self, issue_comment):
+        assert issue_comment._event_resource == 'issue'
+
+    def test_pr(self, pull_request_review):
+        assert pull_request_review._event_resource == 'pull_request'
+
+    def test_other(self):
+        event = GithubWebHookEvent('something_else', None)
+        assert event._event_resource == 'something_else'
+
+
+def test_githubwebhookevent_action(pull_request_review):
+    assert pull_request_review.action == 'submitted'
+
+
+def test_githubwebhookevent_desc(issue_comment):
+    assert issue_comment.description == 'issue_comment: edited'
+
+
+class TestGithubWebHookEventHtmlUrl:
+    def test_issue(self, issue_comment):
+        assert issue_comment.html_url == 'https://example.com/issue/1'
+
+    def test_pr(self, pull_request_review):
+        assert pull_request_review.html_url == 'https://example.com/pull/1'
+
+
+def test_githubwebhookevent_sender_login(issue_comment):
+    assert issue_comment.sender_login == 'issue-sender'
+
+
+class TestGithubWebHookEventUpdatedAt:
+    def test_issue(self, issue_comment):
+        assert (
+            issue_comment.updated_at ==
+            arrow.get('2016-10-24T18:53:10Z').datetime
```

Comment:
To stick with packages we already have, it would be `datetime.datetime(2016, 10, 24, 18, 53, 10, tzinfo=dateutil.tz.tzutc())`